edge/cmd/edgecore/app/server.go

```diff
@@ -51,10 +51,14 @@ offering HTTP client capabilities to components of cloud to reach HTTP servers r
 			// To help debugging, immediately log version
 			klog.Infof("Version: %+v", version.Get())
 
-			// Check running environment before run edge core
-			if err := environmentCheck(); err != nil {
-				klog.Errorf("Failed to check the running environment: %v", err)
-				os.Exit(1)
+			// Check the running environment by default
+			checkEnv := os.Getenv("CHECK_EDGECORE_ENVIRONMENT")
+			if checkEnv == "" || checkEnv == "true" {
```

Comment:
change to `if checkEnv != "false" {    ` ?

Reply:
Done.